kinds/librarypanel/librarypanel_kind.cue

```diff
@@ -0,0 +1,63 @@
+package kind
+
+import "strings"
+
+name:     "LibraryPanel"
+maturity: "experimental"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// 0.0
+			{
+				@grafana(TSVeneer="type")
+
+				// Folder UID
+				folderUid?: string @grafanamaturity(ToMetadata="sys")
+
+				// Library element UID
+				uid: string
+
+				// Panel name (also saved in the model)
+				name: string & strings.MinRunes(1)
+
+				// Panel description
+				description?: string
+
+				// The panel type (from inside the model)
+				type: string & strings.MinRunes(1)
+
+				// Dashboard version when this was saved (zero if unknown)
+				schemaVersion?: uint16
+
+				// panel version, incremented each time the dashboard is updated.
+				version: int64 @grafanamaturity(NeedsExpertReview)
+
+				// TODO: should be the same panel schema defined in dashboard
+				// Typescript: Omit<Panel, 'gridPos' | 'id' | 'libraryPanel'>;
+				model: _
```

Comment:
@sdboyer -- does this seem reasonable as stop gap?

Seems good to have a concrete target for the shared type

Reply:
This works, also could be `{...}`, so it's generated as `Record<string, unknown>` and a bit more defined than just `unknown`.